src/assets/md/e1.md

```diff
@@ -1,7 +1,11 @@
 ```html
```

Comment:
e1 was correct, we do not have here a default value, please rollback

Reply:
I think this default value was added because previous showcase app had it but with latest changes it should be updated as well.